package.json

```diff
@@ -36,6 +36,6 @@
     "redux-actions": "^2.6.5"
   },
   "optionalDependencies": {
-    "@ionic/react": "^5.0.0"
+    "@ionic/react": "^6.4.0"
```

Comment:
Do we have to drop the support for 5?

Reply:
I guess we could do `@ionic/react >= 5.0.0` or something. But this version of the this library will be integrated immediately into the ionic-client-template, which already was upgraded with @ionic/react version V ^6.0.0.

Reply:
Yes, but there are apps built with the older version of the ionic-client-template in the wild, eh? If a user had an app with ^5 and looked at this change, they wouldn't think it was safe to upgrade. In fact, if we don't offer backwards compatibility for ionic, I'd lean towards making it a major upgrade.

But what's the benefit of that? Seems like a hassle to me.

Reply:
Updated to > 5. I also made a few more updates that were changed as part of the client-template update.